INSTALL.md

```diff
@@ -133,6 +133,8 @@ target_link_libraries(foo PRIVATE ${OPENTELEMETRY_CPP_LIBRARIES})
 
 ## Build instructions using Bazel
 
+NOTE: Experimental, and not supported for all the components.
```

Comment:
Probably `STATUS: Experimental,` is better?

Reply:
Yes, it is definitely better. But then we should also be adding the <STATUS> tag rest of the sections of all the docs to indicate whether they are experimental, or stable ( the way done in otel-specs). Would like to avoid doing that :)